code/android-core-library/src/main/java/com/adobe/marketing/mobile/CoreConstants.java

```diff
@@ -11,11 +11,11 @@
 
 package com.adobe.marketing.mobile;
 
-final class CoreConstants {
+public final class CoreConstants {
```

Comment:
Can we revert this file and keep these constants inside Configuration sub package?